spec/models/hackney/note.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+
+describe Hackney::Note do
+  def stub_get_notes
```

Comment:
Don't need to do that now, but just to keep in mind: this and `stub_post_a_note` are used only once, and could be inlined for maximum dumb readability. :shipit:

Reply:
I kind of like it this way, I find the tests more readable.